content/en/providers/check-in/sp/_index.md

```diff
@@ -926,14 +928,14 @@ the API described at the
 This section defines the attributes that can be made available to services
 connected to Check-in.
 
-### 1. EGI ID
-
 <!-- markdownlint-disable line-length no-inline-html -->
 
+### 1. EGI ID
+
 |          attribute name | EGI ID                                                                              |
 | ----------------------: | :---------------------------------------------------------------------------------- |
 |         **description** | An identifier for the user, unique among all EGI accounts and never reused          |
-|   **SAML Attribute(s)** | `1.3.6.1.4.1.5923.1.1.1.13` (eduPersonUniqueId)                                     |
+|   **SAML Attribute(s)** | `urn:oid:1.3.6.1.4.1.25178.4.1.6` (voPersonID)                                      |
```

Comment:
add `1.3.6.1.4.1.5923.1.1.1.13` (eduPersonUniqueId)

Add deprecation note in notes section